backend/scripts/add_unique_ids/add_unique_ids.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import json
+import pathlib
+from typing import Mapping
+
+from sqlalchemy.orm import Session
+
+from app.db.models import Document
+from app.db.session import SessionLocal
+
+
+LookupID = Mapping[str, str]
+
+
+def load_id_map() -> Mapping[str, str]:
+    current_folder = pathlib.Path(__file__).parent.absolute()
+    id_map_path = current_folder / "ids_map.json"
```

Comment:
Where does this file come from? Is it worth adding a README on how it should be used?

Reply:
I really consider this script a one-shot; everything after here should have the IDs and there's no guarantee that what I did this time would continue to work...